packages/page-pattern-modal/src/components/pattern-selector-item.tsx

```diff
@@ -6,25 +6,39 @@ interface PatternSelectorItemProps {
 	onSelect: ( value: string ) => void;
 	patternPostID: number | null;
 	title?: string;
+	stylesheet: string;
```

Comment:
Can we then remove the `theme` props everywhere?

Reply:
The API still keeps using the stylesheet from query parameter, so we cannot remove  it now.

Reply:
No I mean `theme` prop. For example it is not used anymore this file.